data/templates/frr/fabricd.frr.j2

```diff
@@ -0,0 +1,66 @@
+!
+{% if interface is vyos_defined %}
+{%     for iface, iface_config in interface.items() %}
+interface {{ iface }}
+ ip router openfabric VyOS
+ ipv6 router openfabric VyOS
+{%         if iface_config.csnp_interval is vyos_defined %}
+ openfabric csnp-interval {{ iface_config.csnp_interval }}
+{%         endif %}
+{%         if iface_config.hello_interval is vyos_defined %}
+ openfabric hello-interval {{ iface_config.hello_interval }}
+{%         endif %}
+{%         if iface_config.hello_multiplier is vyos_defined %}
+ openfabric hello-multiplier {{ iface_config.hello_multiplier }}
+{%         endif %}
+{%         if iface_config.metric is vyos_defined %}
+ openfabric metric {{ iface_config.metric }}
+{%         endif %}
+{%         if iface_config.passive is vyos_defined %}
+ openfabric passive
+{%         endif %}
+{%         if iface_config.password.md5 is vyos_defined %}
+ openfabric password md5 {{ iface_config.password.md5 }}
+{%         elif iface_config.password.plaintext_password is vyos_defined %}
+ openfabric password clear {{ iface_config.password.plaintext_password }}
+{%         endif %}
+{%         if iface_config.psnp_interval is vyos_defined %}
+ openfabric psnp-interval {{ iface_config.psnp_interval }}
+{%         endif %}
+exit
+{%     endfor %}
+{% endif %}
+!
+router openfabric VyOS
```

Comment:
In FRR it's possible to configure multiple OpenFabric domains. Adding that capability later is going to be more challenging than doing it right now, since we'll need a migration script. I think we should do it right away rather than only allow a single domain with a hardcoded name.